docs/multi-manager-setup.md

```diff
@@ -0,0 +1,157 @@
+# High availability in Swarm
+
+In Docker Swarm, the **swarm manager** is responsible for the entire cluster and manages the resources of multiple `docker` hosts at scale. If the swarm manager dies, you must create a new one and deal with an interruption of service.
+
+The *High Availability* feature of Docker Swarm allows a swarm to handle gracefully the failover of a manager instance. Using this feature, you can create a single **primary manager** instance and multiple **replica** instances.
```

Comment:
I found this sentence a bit hard to read (but I'm not a native English speaker); "allows a swarm to handle gracefully the failover...";

perhaps

```Markdown
The *High Availability* feature allows a Docker Swarm cluster to gracefully handle the failover of a manager instance.
```

I prevented using "a swarm" here, because "swarm" and "cluster" were both used as a term in this document, which could be confusing.